buildSrc/src/main/kotlin/org/cqfn/save/generation/Generation.kt

```diff
@@ -106,37 +107,36 @@ fun generateSavePropertiesClass(jsonObject: Map<String, Option>): TypeSpec.Build
     // Generate secondary ctor
     val secondaryCtor = FunSpec.constructorBuilder()
     secondaryCtor.addParameter("args", ClassName("kotlin", "Array")
-                                                .parameterizedBy(ClassName("kotlin", "String")))
+                                           .parameterizedBy(ClassName("kotlin", "String")))
     secondaryCtor.callThisConstructor()
-    secondaryCtor.addStatement("    val parser = ArgParser(\"save\")")
-                    .addStatement(generateOptions(jsonObject))
-                        .addStatement("    parser.parse(args)")
+    secondaryCtor.addStatement("val parser = ArgParser(\"save\")")
+                    .generateOptions(jsonObject)
+                        .addStatement("parser.parse(args)")
     classBuilder.addFunction(secondaryCtor.build())
     return classBuilder
 }
 
 // Generate options for ArgParser
-fun generateOptions(jsonObject: Map<String, Option>): String {
-    var options = "    "
+fun FunSpec.Builder.generateOptions(jsonObject: Map<String, Option>): FunSpec.Builder {
     jsonObject.forEach {
-        options += "\n${it.key} = parser.option(\n" +
-                "    ${it.value.argType},\n"
-        val fullName = if (it.value.fullName.isNotEmpty()) "    fullName = \"${it.value.fullName}\",\n" else ""
+        var option = "${it.key} = parser.option(\n"
+        option += "${it.value.argType},\n"
+        val fullName = if (it.value.fullName.isNotEmpty()) "fullName = \"${it.value.fullName}\",\n" else ""
```

Comment:
I think we will always have `fullName` for all parameters

Reply:
Yes, I forgot remove this. Previously fullName was optional